api/handler/json_types.go

```diff
@@ -0,0 +1,43 @@
+package handler
+
+import (
+	"encoding/json"
+)
+
+type (
+	stringOrSlice struct {
+		values []string
+	}
+)
+
+func (s *stringOrSlice) UnmarshalJSON(bytes []byte) error {
+	if bytes[0] == '[' {
```

Comment:
`  [....` can be a valid JSON too.

Reply:
Didn't catch your idea. In the case of `[` we have an array. If this array is not a strings array, it is not our problem and there will be an unmarshal error

Reply:
`[...` vs `     [...`, both are valid JSON.

Reply:
Added test with spaces. In case we have the next JSON with spaces:
```json
{    "action"     :      [     "s3:PutObject"     ]     }
```
`bytes` will contain `[     "s3:PutObject"     ]`.
All spaces will be skipped, https://go.dev/src/encoding/json/decode.go:178

Reply:
Updated